src/quickstart/quickstart.md

```diff
@@ -207,10 +211,11 @@ Another more basic way of accessing information, and data, from a request is usi
 app.MapGet("/hello/{name}", (HttpContext ctx) => $"Hello {ctx.Request.RouteValues["name"]}");
```

Comment:
On line 200, there's some broken text and a missing link that I didn't know how to fix.